Makefile

```diff
@@ -769,6 +771,8 @@ bazel-version-dist: bazel-version-dist-linux-amd64 bazel-version-dist-linux-arm6
 	tools/sha256 ${BAZELUPLOAD}/kops/${VERSION}/darwin/amd64/kops ${BAZELUPLOAD}/kops/${VERSION}/darwin/amd64/kops.sha256
 	cp bazel-bin/cmd/kops/windows-amd64/kops ${BAZELUPLOAD}/kops/${VERSION}/windows/amd64/kops.exe
 	tools/sha256 ${BAZELUPLOAD}/kops/${VERSION}/windows/amd64/kops.exe ${BAZELUPLOAD}/kops/${VERSION}/windows/amd64/kops.exe.sha256
+	tar cfvz ${BAZELUPLOAD}/kops/${VERSION}/images/images.tar.gz -C ${BAZELIMAGES} ${BAZELIMAGES}/*.tar.gz ${BAZELIMAGES}/*.tar.gz.sha256
```

Comment:
I don't quite see the point of including the .sha256 of the individual images into the tar file.

Reply:
People may want to validate those individually, or compare to what we have in the GCS bucket.

Reply:
I'm struggling to figure out a valid use case for comparing a .sha256 in a tar file against another file  in the same tar file. What would be the value in validating those individually or comparing to what is in the GCS bucket?

With separate files you can retrieve the small file through a more secure path or use the redundancy to detect cosmic-ray-type errors in the big file. But given the unlikelihood of doing a partial-file download or wanting to salvage part of a slightly corrupted tar file, these use cases don't apply well to checksums embedded in the big file.

Reply:
They don't. There is no real use case, but doesn't break anything either.
if you think it would be best to exclude them, I don't mind doing it.

Reply:
I think it would be best to exclude them.

Reply:
Done.